Tests/BluetoothTests/GATTCharacteristicTests.swift

```diff
@@ -490,6 +491,21 @@ final class GATTCharacteristicTests: XCTestCase {
         XCTAssert(GATTAnalog(data: data) == GATTAnalog(data: data))
     }
     
+    func testBootKeyboardOutputReport() {
+        
+        XCTAssertNil(GATTBootKeyboardOutputReport(data: Data([0x3d, 0x72])))
+        
+        let data = Data([0x01])
+        
+        guard let characteristic = GATTBootKeyboardOutputReport(data: data)
+            else { XCTFail("Could not decode from bytes"); return }
+        
+        XCTAssertEqual(characteristic.data, data)
+        XCTAssertEqual(characteristic.description, "1")
+        XCTAssertEqual(GATTBootKeyboardOutputReport.uuid, .bootKeyboardOutputReport)
+        XCTAssertEqual(GATTBootKeyboardOutputReport(data: data), GATTBootKeyboardOutputReport(data: data))
```

Comment:
Should be:

```
XCTAssertEqual (characteristic, 1)
```

to test for `Equatable` and `ExpressibleByIntegerLiteral`. `.init(data: Data)` is already tested in LOC 500.